### PR TITLE
[Images] Moved limitations from url to new page

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: content/images/image-resizing/format-limitations.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Read JPEG, PNG, GIF (including animations), and WebP images. SVG is not supported, since this format is inherently scalable and does not need resizing.
+- Resize and generate JPEG and PNG images, and optionally AVIF or WebP.
+- Save animations as GIF or animated WebP.
+- Support ICC color profiles in JPEG and PNG images.
+- Preserve JPEG metadata. Metadata of other formats is discarded.
+- Convert the first frame of GIF animations to a still image
+
+AVIF format is supported on a best-effort basis. Images that cannot be compressed as AVIF will be served as WebP instead.
+
+Image Resizing does not apply apply Cloudflare’s Polish feature to resized images, as resized images already have lossy compression applied.
```
Comment (sejoker) [inline content/images/image-resizing/format-limitations.md]:
```suggestion
Image Resizing does not apply Cloudflare’s Polish feature to resized images, as resized images already have lossy compression applied.
```

Comment (sejoker) [reply content/images/image-resizing/format-limitations.md]: where this sentence came from? I would rephrase it to a more positive emphasis:
Image Resizing uses lossy compression by default so  Cloudflare’s Polish feature is applied automatically

Comment (marciocloudflare) [reply content/images/image-resizing/format-limitations.md]: This is actually on our docs in the URL format page, here https://developers.cloudflare.com/images/image-resizing/url-format/#image-optimization-and-interaction-with-polish. Should I delete it? And also that sentence from the URL format page?

Comment (sejoker) [reply content/images/image-resizing/format-limitations.md]: I quite like how IR and Polish is a separate section.  I suggest we move whole paragraph over.

File: content/images/image-resizing/url-format.md [hunk 2]
```diff
@@ -140,10 +128,6 @@ The `fit=scale-down` option ensures that the image will not be enlarged unnecess
 
 You can detect device type by enabling the `CF-Device-Type` header [via Page Rule](https://support.cloudflare.com/hc/articles/229373388).
 
-## Image optimization and interaction with Polish
-
-Polish will not be applied to URLs using Image Resizing. Resized images already have lossy compression applied where possible, so they do not need the optimizations provided by Polish. Use `format=auto` option to allow use of WebP and AVIF formats.
```
Comment (marciocloudflare) [inline content/images/image-resizing/url-format.md:145]: Moved this info to Polish @sejoker

**Review by sejoker — APPROVED** [review]

File: content/images/image-resizing/url-format.md (hunk 2 above)
Comment (sejoker) [reply content/images/image-resizing/url-format.md:145]: looks good.

File: content/images/image-resizing/format-limitations.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+pcx-content-type: reference
+title: Formats and limitations
+meta:
+    title: Supported formats and limitations
+weight: 3
+---
+
+## Supported formats and limitations
```
Comment (pedrosousa) [inline content/images/image-resizing/format-limitations.md]:
```suggestion
# Supported formats and limitations
```

File: content/images/image-resizing/format-limitations.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+Cloudflare Image resizing can:
```
Comment (pedrosousa) [inline content/images/image-resizing/format-limitations.md]:
```suggestion
Cloudflare Image Resizing can:
```

File: content/images/image-resizing/format-limitations.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Read JPEG, PNG, GIF (including animations), and WebP images. SVG is not supported, since this format is inherently scalable and does not need resizing.
+- Resize and generate JPEG and PNG images, and optionally AVIF or WebP.
+- Save animations as GIF or animated WebP.
+- Support ICC color profiles in JPEG and PNG images.
+- Preserve JPEG metadata. Metadata of other formats is discarded.
+- Convert the first frame of GIF animations to a still image
```
Comment (pedrosousa) [inline content/images/image-resizing/format-limitations.md]:
```suggestion
- Convert the first frame of GIF animations to a still image.
```

**Review by pedrosousa — APPROVED** [review]
